spec/features/locales_spec.rb

```diff
@@ -5,7 +5,7 @@
   }
     
   it 'changes to locale in Arabic when Arabic button is clicked' do
-    expect{ find("#change-to-arabic").click }.to change{ I18n.locale }.from(:en).to(:ar)
+    expect{ find("#change-to-arabic").click }.to change{ page.current_path }.from("/en").to("/ar")
```

Comment:
Good! but you could additionally add:

`.and change{ find('html')[:lang] }.from('en').to('ar')`

You can find and example of rspec `.and` in kuina.